changelog.txt

```diff
@@ -1,6 +1,7 @@
 *** WooCommerce Payments Changelog ***
 
 = 3.7.0 - 2022-xx-xx =
+* Add - Disputes list sorting
```

Comment:
Let's rephrase as `Introduce sorting on disputes page.` to align with an entry for filters.

Reply:
Done!